include/drivers/espi_emul.h

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_ESPI_SPI_EMUL_H_
+#define ZEPHYR_INCLUDE_DRIVERS_ESPI_SPI_EMUL_H_
+
+/**
+ * @file
+ *
+ * @brief Public APIs for the eSPI emulation drivers.
+ */
+
+#include <zephyr/types.h>
+#include <device.h>
+
+/**
+ * @brief eSPI Emulation Interface
+ * @defgroup espi_emul_interface eSPI Emulation Interface
+ * @ingroup io_emulators
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define EMUL_ESPI_HOST_CHIPSEL 0
+
+struct espi_emul;
+
+/**
+ * Passes eSPI virtual wires set request to the emulator.
+ * The emulator updates the level of vw.
```

Comment:
what is vw and what is its level?

What does this request actually do? Is it part of the eSPI spec? Can you expand the comments a bit to make it easier to follow for those new to this?

Reply:
Done.